ext/vscode/package.json

```diff
@@ -142,204 +142,197 @@
             }
         },
         "menus": {
-            "explorer/context": [
+            "commandPalette": [
                 {
-                    "when": "resourceFilename =~ /azure.yaml/i",
-                    "command": "azure-dev.commands.cli.provision",
-                    "group": "azure-dev"
+                    "command": "azure-dev.commands.getDotEnvFilePath",
+                    "when": "false"
```

Comment:
Naive question: What is the purpose of listing something here with a `when` clause that is just `false`?

Reply:
VSCode presumes all commands will be shown in the command palette, so in this section, we are explicitly hiding some of them from the palette. In all other places, the commands must be explicitly shown instead of explicitly hidden.